### DISCUSSION TITLE
App Store release

### DISCUSSION THREAD

**Discussion by outsharded** [stackotter/delta-client#62] [Ideas]
Hello, I've got an apple dev license, so if you would like me to release the client on the App Store, [send me an email](mailto:email@tec-kids.co.uk). Also, this should stop the warning of an unknown dev. I cloned the package and I could not get my head round it, so it'll be best if you just send the DMG or IAP file. that is needed. (I can also publish on TestFlight for easier testing if you would like)

**Comment by nekename** [comment]
I think while this would be appreciated in the long term, the client is still quite unstable and not yet ready for release on the platform. We would also have to clarify with the App Store guidelines and with Microsoft copyright before we could proceed.
Although, do check with @stackotter, this is just what I think he would say :)

**Comment by stackotter** [comment]
Sadly, the build tool that Delta Client uses (swift-bundler) does not support signing or sandboxing yet, both of which would be required to go on the App Store. In the future I am planning on getting the client working on iOS, in which case I would definitely get the client working in the sandbox (probably by disabling some non-sandbox-friendly features). But for now, sandboxing isn't the top priority.

In case you're wondering, swift-bundler is used because it solves some problems to do with the plugin system and it has the potential to be more cross platform (potentially adding Linux and Windows support in the future).

There is also possibly the issue of copyright as @ninjadev64 said.

Thanks for the offer though :) I might consider taking it up in the future.
